opentreemap/treemap/js/src/geocoder.js

```diff
@@ -35,12 +35,19 @@ function latLngToParam(latLng) {
 exports = module.exports = function (config) {
 
     var geocodeServer = function (address, magicKey) {
+        var data = {
+            address: address,
+            key: magicKey
+        };
+        if (config.instance.extent) {
```

Comment:
This fails when used on non-instance pages like the `/create` page.
I think it needs to be `if (config.instance && config.instance.extent) {`